### ISSUE TITLE
player.maxArmour is reset every time player dies

### ISSUE THREAD

**Issue by edgarasx** [altmp/altv-issues#1758]
### Description of the problem

After newest gta/altv update player.maxArmour is reset every time player dies

### Reproduction steps

```js
alt.log(player.maxArmour); // output: 100

player.maxArmour = 200;

alt.log(player.maxArmour); // output: 200

// kill player
player.health = 0;

// when player spawns
alt.log(player.maxArmour); // output: 100

```

### Expected behaviour

After players death the maxArmour value should stay the same and not reset.

### Operating system

Windows 10 and Debian 11

### Version

Release 13.45

### Reproduction tested

- [X] I confirm that I have made sure that this issue is also present on the newest dev version

**Comment by FabianTerhorst** [comment]
The .spawn function resets all values.

**Comment by edgarasx** [comment]
> The .spawn function resets all values.

player.maxHealth seems to stay the same after spawn, it's just the maxArmour value that's resetting. But if it's going to stay this way could we receive a alt.on('playerSpawn') server event?

**Comment by litolax** [comment]
Confirmed on the latest release (13.46)
Reproduction steps wihout spawn function:

```js
1) return player.maxArmour; //100

2) 
player.maxArmour = 200;
return player.maxArmour; //200

3) 
player.maxHealth = 300;
return player.maxArmour; //100
```

**Comment by FabianTerhorst** [comment]
Fixed in dev/15.0-dev332
